CHANGELOG.md

```diff
@@ -1,6 +1,14 @@
 # Changelog
 
 ## Unreleased
+### Breaking Changes
+### Upcoming Breaking Changes
+### Additions and Improvements
+- Adds timestamps to enable Prague hardfork on Sepolia and Holesky test networks. [#8163](https://github.com/hyperledger/besu/pull/8163)
```

Comment:
Think we also need to rotate anything into the changelog from the gap since RC1 was cut 
https://github.com/hyperledger/besu/commits/main/CHANGELOG.md

Think it's just this one https://github.com/hyperledger/besu/commit/702ce74c3d6dc95efb77c8c112adc46521663bfa#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4ed